website_compress_html/README.md

```diff
@@ -0,0 +1,27 @@
+Compress HTML Code Output
+=========================
+
+added the option to activate the HTML compession from the master for version 8
+
+you can turn it on or off in the backend
```

Comment:
Please put a section "Configuration" instead explaining full path to configure this.